dspace/config/registries/schema-project-types.xml

```diff
@@ -16,5 +16,28 @@
         - project.identifier.startdate > NOT FOUND
         - project.identifier.expectedcompletion > NOT FOUND
     -->
+	
+	<!--
+	  OpenAIRE4 Guidelines
+      4.2.2. Subproperty funderName (M)
+      Name of the funding provider (occurrence: 1). Mandatory if FundingReference is used. -->
+    <dc-type>
+        <schema>project</schema>
+        <element>funder</element>
+		<element>name</element>
+        <scope_note>Name of the funding provider.</scope_note>
+    </dc-type>
+	
+	<!--
+	  OpenAIRE4 Guidelines
+      4.2.3. Subproperty funderIdentifier (R)
+      Unique identifier of the funding entity (occurrence: 0-1). 
+    -->
+	<dc-type>
+        <schema>project</schema>
+        <element>funder</element>
+		<element>identifier</element>
```

Comment:
<element>funderIdentifier</element>

Reply:
The same I mentioned for `funderName`. Schema.org foresees this field as `project.funder.identifier` 

> The identifier property represents any kind of identifier for any kind of Thing, such as ISBNs, GTIN codes, UUIDs etc. Schema.org provides dedicated properties for representing many of these, either as textual strings or as URL (URI) links. See background notes for more details.